pydis_site/apps/api/models/bot/infraction.py

```diff
@@ -69,6 +69,14 @@ class Infraction(ModelReprMixin, models.Model):
         help_text="Whether a DM was sent to the user when infraction was applied."
     )
 
+    jump_url = models.CharField(
+        default='',
+        max_length=88,
```

Comment:
This is pretty tight, I can't think of any valid reason it should be longer than this so I guess it's fine though...